tests/test_explicit.py

```diff
@@ -0,0 +1,69 @@
+from math import isclose
+
+import pytest
+from firedrake import *
+from irksome import PEPRK, Dt, MeshConstant, TimeStepper
+from ufl.algorithms.ad import expand_derivatives
+
+peprks = [PEPRK(*x) for x in ((4, 2, 5), (5, 2, 6))]
+
+
+@pytest.mark.parametrize("butcher_tableau", peprks)
+def test_1d_heat_dirichletbc(butcher_tableau):
```

Comment:
A test is a test, but users do also sometimes read tests for go-by in code usage. Should we be doing a wave equation so as not to give the wrong idea about applicability of explicit methods for the heat equation?

Reply:
This gets complicated, since the mixed wave equation form wouldn't strongly impose Dirichlet BCs on the solution.  So, I opted to copy a test that we were already using to test BC enforcement (from `test_dirichletbc.py`), and that happened to be heat equation.

I think this boils down to mechanism over policy, in any case

Reply:
Agreed, one could do the wave equation in H^1 to get strongly enforced BC (or use reflecting BC in mixed form), but this is a test.  Maybe a comment saying “this is only a test of BC feature.  Please don’t actually use explicit methods on the heat equation and then complain to the Irksome developers that our code is busted and gives massive instabilities.

Reply:
Warning label added...